### ISSUE TITLE
MONTH and DAY formulas function differently than lesson shows

### ISSUE THREAD

**Issue by ndporter** [datacarpentry/spreadsheets-socialsci#153]
In the [Dates episode](https://datacarpentry.org/spreadsheets-socialsci/03-dates-as-data/index.html), in the current version of Excel for Office 365 (both Windows and Mac), the formula `=MONTH(A2)` in the `MM_DD_YEAR` sheet returns a `Date` formatted cell that displays as `1/11/1900` unless the column with the formula has been set to Numeric format. Likewise, `=DAY(A2)` shows as `1/17/1900`.

I don't think this was default behavior in older versions but it was consistent for my whole 7-person workshop.

I would recommend playing around for a more reliable way to do that formula or else expanding with a note about changing the display format.

**Comment by josenino95** [comment]
Taking a fresh look at this: I think the display format of the new cells (where you input the `=MONTH()` or the other functions) depend on where you create the cell.

I tried it out and I think cells created after and Insert column operation "inherit" the display format of the column to the left.

To show you this, here is a screenshot of my little trial. I just created the new values, I didn't change any display formatting

<img width="1191" height="437" alt="Image" src="https://github.com/user-attachments/assets/b3ec8da9-7a45-4230-855e-632d7c54b139" />

The B column is what Excel displays if I inserted a a column there, and it inherits the "Date" display format of column A.

The I column was previously empty, and I see it just has the "General" display format, so it successfully returns the corresponding number.

In conclusion, I think this clarification of where to create the day, month, and year columns could be made in a callout or a teaching note in the lesson.
